testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeDatabend.java

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Thanks for adding a product test. Please also update `EnvMultinodeAllConnectors`.

Reply:
Done.